docs/resources/acc_crypto_policy.md

```diff
@@ -17,40 +17,22 @@ variable "acct_id" {
   type    = string
   default = "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX"
 }
-// Add cryptographic policy to a Fortanix DSM account
+## Add cryptographic policy to a Fortanix DSM account
+
+# This resource is an example of a crypto policy with all the permissions allowed.
 resource "dsm_acc_crypto_policy" "name" {
   acct_id = var.acct_id
   cryptographic_policy = jsonencode({
-    legacy_policy = "allowed"
+    legacy_policy = "allowed" # other values: prohibited and unprotect_only
```

Comment:
Change the wording of the comment to something like:
```
# other accepted values: `"prohibited"` or `"unprotect_only"`
```
And is `unproctect_only` correct?

Reply:
Yes, it is unprotect_only.

And updated.